src/utils/dateForm.ts

```diff
@@ -0,0 +1,61 @@
+export const chatBubbleTime = (chatDate: string) => {
+  const date = new Date(chatDate);
+  const hours = date.getHours();
+  const minutes = date.getMinutes();
+  const isAM = hours < 12 ? true : false;
+  const hours12 = hours < 13 ? hours : hours - 12;
+  return `${isAM ? "오전" : "오후"} ${hours12}:${
+    minutes < 10 ? "0" + minutes : minutes
+  }`;
```

Comment:
```suggestion
  const date = new Date(chatDate);
  const hours = date.getHours();
  const minutes = date.getMinutes();
  const isAM = hours < 12;
  const hours12 = hours < 13 ? hours : hours - 12;
  
  const formattedTime = `${isAM ? "오전" : "오후"} ${hours12}:${minutes < 10 ? "0" + minutes : minutes}`;
  return formattedTime;
```
여기는 이렇게 isAM 에 대해선 `hours>12`가 이미 boolean 값이고, 마지막엔 가독성 측면에서 좀 수정해봤습니다!!!